soc/arm/nxp_imx/rt/Kconfig.defconfig.mimxrt1024

```diff
@@ -0,0 +1,26 @@
+# i.MX RT1024
+
+# Copyright (c) 2020, NXP
```

Comment:
This is definitely your file and your work. Put your copyright here. And in all all other your files.